internal/bminventory/inventory.go

```diff
@@ -2805,9 +2805,8 @@ func shouldHandle(params installer.PostStepReplyParams) bool {
 }
 
 func (b *bareMetalInventory) PostStepReply(ctx context.Context, params installer.PostStepReplyParams) middleware.Responder {
-	log := logutil.FromContext(ctx, b.log)
-	msg := fmt.Sprintf("Received step reply <%s> from cluster <%s> host <%s>  exit-code <%d> stdout <%s> stderr <%s>", params.Reply.StepID, params.ClusterID,
-		params.HostID, params.Reply.ExitCode, params.Reply.Output, params.Reply.Error)
+	log := logutil.EntryFromContext(ctx, b.log)
```

Comment:
Why not use logutil.FromContext like all other places?

Reply:
FromContext does not support testing agains the actual log level

Reply:
I think there is no problem to write both in debug messages and info messages.

Reply:
sure, but the message will be logged twice

Reply:
It is debug.  We can allow it

Reply:
log.Debugf("%s stdout <%s> ", sb.String(), params.Reply.Output)
log.Info(sb.String())
is this code better?